### ISSUE TITLE
shaderc make step fails

### ISSUE THREAD

**Issue by rld243** [jb-alvarado/media-autobuild_suite#669]
Thanks for fixing #667 but now I've run into another problem:

```
ffmpeg git ................................................ [Up-to-date]
┌ angle-latest-win64 ................................. [File up-to-date]
├ Running extract...
  Running git update for mujs...
mujs git .................................................. [Up-to-date]
  Running git update for vulkan...
vulkan git ................................................ [Up-to-date]
  Running git update for shaderc...
┌ shaderc git .......................................... [Files missing]
├ Running glslang-reset...
├ Running glslang-pull...
├ Running spirv-tools-reset...
├ Running spirv-tools-pull...
├ Running spirv-headers-reset...
├ Running spirv-headers-pull...
├ Running cmake...
├ Running make...
Likely error:
[2/224] Update build-version.inc in the Shaderc build directory (if necessary).
[3/224] cmd.exe /C "cd /D D:\Tools\mingw\build\shaderc-git\build-64bit\third_party\spirv-tools && cmake -DCHANGES_FILE=D:/Tools/mingw/build/shaderc-git/third_party/spirv-tools/CHANGES -DTEMPLATE_FILE=D:/Tools/mingw/build/shaderc-git/third_party/spirv-tools/cmake/SPIRV-Tools.pc.in -DOUT_FILE=D:/Tools/mingw/build/shaderc-git/build-64bit/third_party/spirv-tools/SPIRV-Tools.pc -DCMAKE_INSTALL_PREFIX=D:/Tools/mingw/local64 "-DSPIRV_LIBRARIES=-lSPIRV-Tools -lSPIRV-Tools-link -lSPIRV-Tools-opt" -P D:/Tools/mingw/build/shaderc-git/third_party/spirv-tools/cmake/write_pkg_config.cmake"
[4/224] Generate info tables for GLSL extended instructions and operands v1.2.
FAILED: third_party/spirv-tools/glsl.std.450.insts.inc
cmd.exe /C "cd /D D:\Tools\mingw\build\shaderc-git\build-64bit\third_party\spirv-tools\source && D:\Tools\mingw\msys64\usr\bin\python.exe D:/Tools/mingw/build/shaderc-git/third_party/spirv-tools/utils/generate_grammar_tables.py --extinst-glsl-grammar=D:/Tools/mingw/build/shaderc-git/third_party/spirv-headers/include/spirv/1.2/extinst.glsl.std.450.grammar.json --glsl-insts-output=D:/Tools/mingw/build/shaderc-git/build-64bit/third_party/spirv-tools/glsl.std.450.insts.inc"
  File "D:/Tools/mingw/build/shaderc-git/third_party/spirv-tools/utils/generate_grammar_tables.py", line 695
    if args.core_insts_output is not None:
                                         ^
TabError: inconsistent use of tabs and spaces in indentation
ninja: build stopped: subcommand failed.
```
[logs.zip](https://0x0.st/s8mA.zip)

**Comment by rld243** [comment]
Thanks. Yet another problem occurs:
```
┌ shaderc git ....................................... [Recently updated]
├ Running uninstall...
├ Running glslang-clone...
├ Running spirv-tools-clone...
├ Running spirv-headers-clone...
├ Running cmake...
├ Running make...
├ /local64/lib/libshaderc_shared.a ......................... [Not found]
└ shaderc-git ................................................. [Failed]

Try deleting '/build/shaderc-git' and start the script again.
If you're sure there are no dependencies <Enter> to continue building.
Close this window if you wish to stop building.
```
I tried to delete the folder, didn't help, same error.
If I press Enter to continue building, mpv compiles without any errors, so I guess it's ok to ignore this warning.

**Comment by wiiaboo** [comment]
Fixed in dbdc13d
